client/src/frontend/components/journal/VolumeDetail/styles.js

```diff
@@ -0,0 +1,25 @@
+import styled from "@emotion/styled";
+import { fluidScale } from "theme/styles/mixins";
+
+export const Wrapper = styled.div`
+  padding-block-start: var(--container-padding-block-start);
+  padding-block-end: var(--container-padding-block-end);
+`;
+
+export const IssueWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
```

Comment:
Fwiw, we ran into issues using `gap` with `flex` on NGLP in Safari <= 14.0, and ended up deciding not to use it for now.

Reply:
Boo Safari 👎🏻. Since it's fixed in 14.5, I'm inclined to keep it rather than revert to the old negative margin hack. That's two major iOS versions we'd be supporting. Happy to discuss with the larger group though. It raises a larger question of how the team should handle Safari (and all iOS browsers), since it's very frustratingly tethered to the OS version. If we want to support older releases, we'll want to take a closer look at what we can/can't use in CSS and JS. I know, for instance, that logical properties have only partial support in Safari 13.